integration_test/run.sh

```diff
@@ -6,7 +6,7 @@ rm -rf ${TESTDIR}
 mkdir -p ${TESTDIR}/1 ${TESTDIR}/2
 
 # To kill any remaining open bitcoind.
-killall -9 bitcoind
+# killall -9 bitcoind
```

Comment:
I'd prefer to either completely delete this line, leave it alone or make it optional through some flag (depending on if it's actually doing something useful in some CI context).

Reply:
Maybe @stevenroose could clarify if it was needed for CI

Reply:
Also, FYI I made a dev-dep only small crate to handle bitcoind processes https://github.com/RCasatta/bitcoind with some features like finding a free port and always killing the process no matter how the test ends.
I think it's more maintainable than a shell script
(I also just added a small PR https://github.com/RCasatta/bitcoind/pull/3 to allow creating connected p2p nodes to support this repo testing env)

Reply:
killall removed

Reply:
Any reason we have not stated to use your `bitcoind` crate `RCassata`? I'll write up an issue if not.